README.md

```diff
@@ -54,7 +54,9 @@ This is equivalent to:
 ```lua
 require("lsp-file-operations").setup {
   -- used to see debug logs in file `vim.fn.stdpath("cache") .. lsp-file-operations.log`
-  debug = false
+  debug = false,
+  -- how long to wait for file rename information before cancelling
+  timeout = 10000,
```

Comment:
in general when working with time I find it a good practice to name variables like `timeout_ms` or something like this. So it would be easier to configure without looking to docs.

Reply:
Good idea, that's been updated.